x-pack/legacy/plugins/ml/public/services/ml_api_service/index.d.ts

```diff
@@ -47,14 +48,10 @@ declare interface Ml {
     getDataFrameTransforms(jobId?: string): Promise<any>;
     getDataFrameTransformsStats(jobId?: string): Promise<any>;
     createDataFrameTransform(jobId: string, jobConfig: any): Promise<any>;
-    deleteDataFrameTransform(jobId: string): Promise<any>;
+    deleteDataFrameTransforms(jobsData: DataFrameTransformEndpointData[]): Promise<any>;
```

Comment:
these return `Promise<DataFrameTransformEndpointResultData>`

Reply:
Updated in f7a9d5de55631c0e03cb43586efbddc0b2a15555